admin/admin-service/README.md

```diff
@@ -1,11 +1,28 @@
 # Admin Service
 Admin Service can be accessed only by the privileged group of admin personnel, its used to perform bulk data operation.
 
+## Build & run (for developers)
+The project requires JDK 21.0.3
+and mvn version - 3.9.6
+1. Build and install:
+    ```
+    $ cd kernel
+    $ mvn install -DskipTests=true -Dmaven.javadoc.skip=true -Dgpg.skip=true
+    ```
+
 ## Configuration files
-Admin Service uses the following configuration files:
+Admin Service uses the following configuration files that are accessible in this [repository](https://github.com/mosip/mosip-config/tree/master).
```

Comment:
Please add this line.
We are using two property files that are accessible in this [repository](https://github.com/mosip/mosip-config). Please refer the required released tagged version for configuration.